src/visualisations/gender-disparity/js/employment_main.js

```diff
@@ -0,0 +1,29 @@
+import Chart from './chart.js'
+import Search from './search.js'
+import Toggle from './toggle.js'
+import * as data from '/data/muni-unemployment-education.json'
```

Comment:
this is the same dataset as https://github.com/OpenUpSA/OpenUp-data-viz/pull/7/files#diff-1befe1b56586a1a2ffe2fd56a19635f31903bf9c78d9ef76ac9d2c104810eaabR4 - isn't the point that it should be different?

Reply:
Hi JD, yeah this dataset has both the education and employment data. So I'm using it for both viz